app/views/steps/claim_details/edit.html.erb

```diff
@@ -15,6 +15,12 @@
             <% if field == :preparation_time %>
               <%= f.govuk_period_field :time_spent, width: 'one-third' %>
             <% end %>
+            <% if field == :work_before %>
+             <%= f.govuk_date_field :work_before_date, width: 'one-third' %>
+            <% end %>
+            <% if field == :work_after %>
+             <%= f.govuk_date_field :work_after_date, width: 'one-third' %>
+            <% end %>
```

Comment:
can we try an cleanup the code in this loop...

```ruby
        <%= f.govuk_radio_buttons_fieldset field, legend: { size: 'm' } do %>
          <%= f.govuk_radio_button field, YesNoAnswer::YES.to_s do%>
            <%= 
                     if field == :preparation_time
                       f.govuk_period_field :time_spent, width: 'one-third'
                     elsif %i[work_before work_after].include?(field)
                       f.govuk_date_field "#{field}_date", width: 'one-third' %>
                     end
            %>
          <%= f.govuk_radio_button field, YesNoAnswer::NO.to_s %>
        <% end%>
```
the goal here is to get something that it easy to read for a dev when they come in...

> NOTE: Happy for you to do this in a different way, just not what you had. :)

Reply:
This reads ok to me - my only confusion is the condition in line 18. Could this be a helper method that takes the field and the name of the method describes the intention of it?